packages/graphql-language-service/package.json

```diff
@@ -29,11 +29,11 @@
     "prepublish": "node ../../resources/prepublish.js"
   },
   "peerDependencies": {
-    "graphql": "^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0"
+    "graphql": "^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^0.14.0"
```

Comment:
You mean `^14.0.0` here and elsewhere in this PR - GraphQL jumped from `0.13` to `14.0`.

Reply:
thanks for that, just saw you review the same issue in a `graphiql` PR yesterday